### ISSUE TITLE
Terrain data does not show

### ISSUE THREAD

**Issue by 1orbit** [nebriv/VTOLVR-TacviewLogger#22]
Have followed terrain instructions but terrain will not display. CustomHeightmapList.xml is shown below (VTOL terrain data is the bottom entry):
https://pastebin.com/rQgUz9nb

**Comment by nebriv** [comment]
Hi, could you provide your player log and what map you were testing on?

**Comment by Noxium** [comment]
Bumping this as I have the same issue. I did all the steps as listed but I still only see the default terrains listed (shown below). This is my first time using TacView so it's possible this is user error on my part.

- C:\ProgramData\Tacview\Data\Terrain\Custom\CustomHeightmapList.xml : https://pastebin.com/6YXE8n6x
- Map: The Island - Included at C:\ProgramData\Tacview\Data\Terrain\Custom\heightmap_TheIsland.data
- Datalog (I kinda got tunnel vision at the end there lol) : [datalog.acmi.txt](https://github.com/nebriv/VTOLVR-TacviewLogger/files/7167619/datalog.acmi.txt)

![image](https://user-images.githubusercontent.com/12095464/133385995-dd77db05-1d93-4994-8444-d5e1ee657076.png)

**Comment by Noxium** [comment]
Actually I think the terrain is importing properly, but the offset is off. This is a zoomed out version of the mission 'Counter-Bomber' for the F45, the way it's unnaturally cut-out confirms that this is definitely artificial terrain.

![image](https://user-images.githubusercontent.com/12095464/133718217-079d0cf7-14b2-4969-a3f0-e959fc7dea6d.png)

However, there's still an issue as I'm occasionally flying through terrain. This is me landing at the FARP on the same mission.

![image](https://user-images.githubusercontent.com/12095464/133719095-71e8797b-26ec-44ba-aa42-188bc8fac18c.png)

Some kind of offset issue?

**Comment by nebriv** [comment]
Yeah this sounds about right. I just randomly guessed on a terrain offset. You can edit the XML to find a better fit. 

These are the values I went with:
altFactor = 0.04f;
altOffset = -235f;

> Bumping this as I have the same issue. I did all the steps as listed but I still only see the default terrains listed (shown below). This is my first time using TacView so it's possible this is user error on my part.
> 
>     * C:\ProgramData\Tacview\Data\Terrain\Custom\CustomHeightmapList.xml : https://pastebin.com/6YXE8n6x
> 
>     * Map: The Island - Included at C:\ProgramData\Tacview\Data\Terrain\Custom\heightmap_TheIsland.data
> 
>     * Datalog (I kinda got tunnel vision at the end there lol) : [datalog.acmi.txt](https://github.com/nebriv/VTOLVR-TacviewLogger/files/7167619/datalog.acmi.txt)
> 
> 
> ![image](https://user-images.githubusercontent.com/12095464/133385995-dd77db05-1d93-4994-8444-d5e1ee657076.png)

Yep - annoyingly theres no way to _just_ import the vtol heightmaps. Tacview wants to exclusively use the world, and then you can add overlays on top of it. So I've basically just shoved a heightmap into the middle of the ocean.
